bin/common/usr_local_bin_to_path.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Add /usr/local/bin to non-login path, since tools are installed there
+if ! grep ENV_SUPATH /etc/login.defs | grep -q /usr/local/bin; then
+  sed -i '/ENV_SUPATH/s/$/:\/usr\/local\/bin/' /etc/login.defs
```

Comment:
Right, I thought `/etc/environment.d/` is done via `pam_env`, which should be on login… I'm assuming by _non-login_ you mean _non-interactive_ here, though.  Not sure how else you'd get different privileges otherwise (since IIRC vagrant privileged shells are done via `sudo`)